src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -93,6 +96,10 @@ public function newAction(Request $request)
             // See http://symfony.com/doc/current/book/controller.html#flash-messages
             $this->addFlash('success', 'post.created_successfully');
 
+            if ($form['createAndShow']->isClicked()) {
```

Comment:
this `$form['createAndShow']` is not wrong, but perhaps could bring confusion, rather use `$form->get('createAndShow')`: I have seen it used in the documentation always.